src/components/content-tab-import-export.tsx

```diff
@@ -21,13 +21,18 @@ interface ContentTabImportExportProps {
 	selectedSite: SiteDetails;
 }
 
+const siteSyncingMessage = __(
```

Comment:
As the message is defined outside the component, will translations work?

Reply:
They won't unfortunately. Great catch!

I wanted to prevent the duplication of that string not realizing it won't work with translations. I have now updated the implementation in the latest commit (https://github.com/Automattic/studio/pull/717/commits/8aaffedafcb4231f3a2b1e83e58506bc0a00cfce).

Feel free to let me know what you think. Thank you.